terraform/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "region" {
+  default = "us-west-1"
+}
+
+variable "account" {
+  default = ""
+}
+
+#########
+# Redis #
+#########
+
+variable "redis-instance" {
+  default = ""
+}
+
+variable "redis-num-nodes" {
+  default = 1
+}
+
+variable "redis-version" {
+  default = "5.0.4"
+}
+
+##########
+#  PSQL  #
+##########
+
+variable "psql-instance" {
+  default = ""
```

Comment:
Same as above, recommend you set a default value here too and no leave it to `""`

Reply:
thanks, that's useful input